packages/plugin-git/sources/gitUtils.ts

```diff
@@ -0,0 +1,26 @@
+import {execUtils}                                    from '@berry/core';
+import {xfs, NodeFS, PortablePath, ppath, Filename}   from '@berry/fslib';
+
+const tmp = require('tmp');
+
+export async function clone(repo: string) {
+  const directory = await createTemporaryFolder()
+  const {code, stdout} = await execUtils.execvp(`git`, [`clone`, `${repo}`, `${directory}`], {cwd: directory});
```

Comment:
@arcanis any suggestions how we should handle the output of `execvp` command? 🤔 Throwing and error? 🤔

Reply:
`execvp` has a `strict: true` option for that! 🙂